spec/ddoc.src.dd

```diff
@@ -685,104 +685,7 @@ $(P
-DDOC_AUTHORS          = $(DOLLAR)(B Authors:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)$(DOLLAR)(LF)
-DDOC_BUGS             = $(DOLLAR)(RED BUGS:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)$(DOLLAR)(LF)
-DDOC_COPYRIGHT        = $(DOLLAR)(B Copyright:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)$(DOLLAR)(LF)
-DDOC_DATE             = $(DOLLAR)(B Date:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)$(DOLLAR)(LF)
-DDOC_DEPRECATED       = $(DOLLAR)(RED Deprecated:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)$(DOLLAR)(LF)
-DDOC_EXAMPLES         = $(DOLLAR)(B Examples:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)$(DOLLAR)(LF)
-DDOC_HISTORY          = $(DOLLAR)(B History:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)$(DOLLAR)(LF)
-DDOC_LICENSE          = $(DOLLAR)(B License:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)$(DOLLAR)(LF)
-DDOC_RETURNS          = $(DOLLAR)(B Returns:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)$(DOLLAR)(LF)
-DDOC_SEE_ALSO         = $(DOLLAR)(B See Also:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)$(DOLLAR)(LF)
-DDOC_STANDARDS        = $(DOLLAR)(B Standards:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)$(DOLLAR)(LF)
-DDOC_THROWS           = $(DOLLAR)(B Throws:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)$(DOLLAR)(LF)
-DDOC_VERSION          = $(DOLLAR)(B Version:)$(DOLLAR)(BR)$(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)$(DOLLAR)(LF)
-DDOC_SECTION_H        = $(DOLLAR)(B $(DOLLAR)0:)$(DOLLAR)(BR)$(DOLLAR)(BR)$(DOLLAR)(LF)
-DDOC_SECTION          = $(DOLLAR)0$(DOLLAR)(BR)$(DOLLAR)(BR)$(DOLLAR)(LF)
-DDOC_MEMBERS          = $(DOLLAR)(DL $(DOLLAR)0)
-DDOC_MODULE_MEMBERS   = $(DOLLAR)(DDOC_MEMBERS $(DOLLAR)0)$(DOLLAR)(LF)
-DDOC_CLASS_MEMBERS    = $(DOLLAR)(DDOC_MEMBERS $(DOLLAR)0)$(DOLLAR)(LF)
-DDOC_STRUCT_MEMBERS   = $(DOLLAR)(DDOC_MEMBERS $(DOLLAR)0)$(DOLLAR)(LF)
-DDOC_ENUM_MEMBERS     = $(DOLLAR)(DDOC_MEMBERS $(DOLLAR)0)$(DOLLAR)(LF)
-DDOC_TEMPLATE_MEMBERS = $(DOLLAR)(DDOC_MEMBERS $(DOLLAR)0)$(DOLLAR)(LF)
-DDOC_ENUM_BASETYPE    = $(DOLLAR)0
-DDOC_PARAMS           = $(DOLLAR)(B Params:)$(DOLLAR)(BR)\n$(DOLLAR)(TABLE $(DOLLAR)0)$(DOLLAR)(BR)$(LF)
-DDOC_PARAM_ROW        = $(DOLLAR)(TR $(DOLLAR)0)$(DOLLAR)(LF)
-DDOC_PARAM_ID         = $(DOLLAR)(TD $(DOLLAR)0)$(DOLLAR)(LF)
-DDOC_PARAM_DESC       = $(DOLLAR)(TD $(DOLLAR)0)
-DDOC_BLANKLINE        = $(DOLLAR)(BR)$(DOLLAR)(BR)
-
-DDOC_ANCHOR           = $(LT)a name="$(DOLLAR)1"$(GT)$(LT)/a$(GT)
-DDOC_PSYMBOL          = $(DOLLAR)(U $(DOLLAR)0)
-DDOC_PSUPER_SYMBOL    = $(DOLLAR)(U $(DOLLAR)0)
-DDOC_KEYWORD          = $(DOLLAR)(B $(DOLLAR)0)
-DDOC_PARAM            = $(DOLLAR)(I $(DOLLAR)0)
-DDOC_CONSTRAINT         = $(DDOC_CONSTRAINT) if ($0)
-DDOC_OVERLOAD_SEPARATOR =
-DDOC_TEMPLATE_PARAM_LIST= $0
-DDOC_TEMPLATE_PARAM     = $0
-DDOC_BACKQUOTED       = $(DOLLAR)(D_INLINECODE $(DOLLAR)0)
-
-ESCAPES = /$(LT)/$(AMP)lt;/
-          /$(GT)/$(AMP)gt;/
-          /$(AMP)/$(AMP)amp;/
+3fc0f638fb1ab1bad2e01f507b968324
```

Comment:
I generated it with mcookie. The idea is it's a unique string to be replaced with the actual text.